pkg/commands/build/get_command.go

```diff
@@ -22,23 +22,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// AddCommandsTo adds commands to a root cobra command.
-func AddCommandsTo(ctx context.Context, fio files.FileReaderWriter, sio cmdlib.StdioReaderWriter, root *cobra.Command) {
+func GetCommand(ctx context.Context, fio files.FileReaderWriter, sio cmdlib.StdioReaderWriter, gopts *cmdlib.GlobalOptions) *cobra.Command{
```

Comment:
lost method comment

Reply:
Can you explain why we want comments for these type of functions? 
I removed it intentionally.

Reply:
It'll ask for that in the lint because these are public functions. So, we should corresponding comments. you can run /hack/check-lint.sh or whatever it's called